src/application/redux/reducers/utxos-unspents.ts

```diff
@@ -0,0 +1,277 @@
+import type { NetworkString, UnblindedOutput } from 'ldk';
+import { crypto } from 'ldk';
+import type { AccountID } from 'marina-provider';
+import { MainAccountID } from '../../../domain/account';
+import type { TxDisplayInterface } from '../../../domain/transaction';
+import { TxStatusEnum } from '../../../domain/transaction';
+import type { UtxosTransactionsState } from '../../../domain/wallet';
+import { toStringOutpoint } from '../../utils/utxos';
+import * as ACTION_TYPES from '../actions/action-types';
+
+// minimum time utxos are locked (5 minutes)
+const lockedUtxoMinimunTime = 300_000;
```

Comment:
```suggestion
const lockedUtxoMinimumTime = 300_000;
```
And do a find & replace project wide